Source/XUnitExtensions/Lit/MainMethodLitCommand.cs

```diff
@@ -34,6 +34,10 @@ public class MainMethodLitCommand : ILitCommand {
 
       var exitCode = (int)assembly.EntryPoint!.Invoke(null, new object[] { arguments })!;
 
+      inputReader?.Close();
```

Comment:
Shouldn't we do this at the call site of `Execute`, where the disposables are created?

Reply:
I'll wait for @robin-aws  to answer this one because he wrote this part.

Reply:
You're closing at the `Execute` call site now so these lines can be removed